runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/CoderTranslationTest.java

```diff
@@ -104,8 +104,8 @@ public void validateKnownCoders() {
     public void validateCoderTranslators() {
       assertThat(
           "Every Known Coder must have a Known Translator",
-          CoderTranslation.KNOWN_CODER_URNS.keySet(),
-          equalTo(CoderTranslation.KNOWN_TRANSLATORS.keySet()));
+          ModelCoderRegistrar.BEAM_MODEL_CODERS.keySet(),
```

Comment:
should this be `ModelCoderRegistrar. BEAM_MODEL_CODER_URNS.keySet()` ?

Reply:
They're the same set of keys, so it doesn't really matter